src/Mod/Sketcher/Gui/DrawSketchDefaultWidgetHandler.h

```diff
@@ -989,7 +989,7 @@ class DrawSketchDefaultWidgetHandler: public DrawSketchDefaultHandler<HandlerT,
         }
 
         /* This function give the focus to a spinbox and register to which it gave it.*/
-        void setFocusToLabel(int labelindex)
+        void setFocusToLabel(size_t labelindex)

```

Comment:
size_t should not be used as type of parameter, unless one really means a size. Here it is an index, not a size.

to solve the iwarning there are two options. If no negative indices are envisaged, then the parameter can be made unsigned. If negative indices are envisaged, then the changes are within the function, you may static_cast the size_t to int, considering that the index will never go to the overflow region by design.

Reply:
I make no changes for now right ?

Reply:
Right. I will come back to you with another branch, and we discussed it from there. 

It is not worth that you invest time in rebasing and correcting this branch, as I need to do it in parallel to add some other changes. 

I am leaving the comments in this PR, as it is convenient so that we understand each other.